tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -0,0 +1,211 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: Saying hello!,
+    Resource: SayHello,
+    Service: MyServiceName,
+    Type: custom,
+    Tags: {
+      attribute-bool: true,
+      attribute-boolArray: [false,true,false],
+      attribute-boolArrayEmpty: [],
+      attribute-doubleArray: [4.1,5,6],
+      attribute-doubleArrayEmpty: [],
+      attribute-intArray: [1,2,3],
+      attribute-intArrayEmpty: [],
+      attribute-string: str,
+      attribute-stringArray: ["str1","str2","str3"],
+      attribute-stringArrayEmpty: [],
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: MyServiceName,
+      otel.status_code: STATUS_CODE_UNSET,
```

Comment:
if unset, shall we add it everytime? I assume it's a spec somewhere I haven't read, just asking.

Reply:
There is a native Status field for OTLP spans, as shown here: https://github.com/open-telemetry/opentelemetry-proto/blob/d8729d40f629dba12694b44c4c32c1eab109b00a/opentelemetry/proto/trace/v1/trace.proto#L268

For the scenario where an app sends an OTLP trace to the Datadog trace agent, it's always set so the `otel.status_code` always gets set before sending to the backend. That's why I'm always setting the `otel.status_code` tag when we create Datadog spans